### PR TITLE
MVP

### PR DESCRIPTION
**Pull request by ajb85**
MVP Complete, moving to stretch

### CODE REVIEW

**Comment by ajb85** [comment]
Stretch complete

File: enhancing/enhancer.js [hunk 1]
```diff
@@ -2,21 +2,43 @@ module.exports = {
   succeed,
   fail,
   repair,
-  get,
+  get
 };
-
 function succeed(item) {
+  // enhancement++ ✓
```
Comment (GannonDetroit) [inline enhancing/enhancer.js:8]: good job with your success and failure functions

File: enhancing/enhancer.js [hunk 2]
```diff
@@ -2,21 +2,43 @@ module.exports = {
...
+  } else {
+    return {
+      ...item,
+      durability: item.durability - 10,
+      enhancement: enhancement - 1
+    };
+  }
 }
 
 function repair(item) {
```
Comment (GannonDetroit) [inline enhancing/enhancer.js:36]: good job with the repair and get functions too

File: enhancing/enhancer.spec.js [hunk 3]
```diff
@@ -1,2 +1,62 @@
-const enhancer = require('./enhancer.js');
+const { succeed, fail, repair, get } = require("./enhancer.js");
 // test away!
+
+const item = { name: "Very Real Item", durability: 59, enhancement: 4 };
+
```
Comment (GannonDetroit) [inline enhancing/enhancer.spec.js:5]: good job testing your functions, I like how you added multiple describes within this describe to better highlight the purpose of each test.

**Review by GannonDetroit — COMMENTED** [review]
excellent job with your first tests.

**Pull request merged**